LibHR/Geometry/new_geom_gpu.hpp

```diff
@@ -1,38 +0,0 @@
-//TODO this code is chaotic and needs better integration with the concepts in the
-//     new geometry.
-
-#define _DECLARE_KERNEL(_name, _type, _size) \
```

Comment:
Why do you need a separate file for this? you can include this code in new_geom_gpu.cu